core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/environment/EnvironmentsController.java

```diff
@@ -66,6 +65,20 @@
         return (PagedModel<EnvironmentDto>) environmentDtoPagedResourcesAssembler.toEmptyModel(environmentDtoPage, EnvironmentDto.class);
     }
 
+    @GetMapping("/list")
+    @PreAuthorize("hasPrivilege('ENVIRONMENTS_READ')")
+    public HalMultipleEmbeddedResource<EnvironmentDto> getAll() {
+        List<EnvironmentDto> environmentDtos = environmentDtoService.getAll();
```

Comment:
can't you call the public Page<EnvironmentDto> getAll(Pageable pageable) and provide the Pageable.unpaged() as an argument?